data_steward/validation/metrics/required_labs.py

```diff
@@ -54,6 +80,67 @@ def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
     :return: None
     """
 
+    descendants_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+
+    client = bq.get_client(project_id)
+    dataset = client.dataset(dataset_id)
+    vocab_dataset = client.dataset(common.VOCABULARY_DATASET)
+
+    # concept table and concept ancestor table source tables
+    concept_source_table = vocab_dataset.table(common.CONCEPT)
+    concept_ancestor_source_table = vocab_dataset.table(common.CONCEPT_ANCESTOR)
+
+    # concept table and concept ancestor table destination tables
+    concept_dest_table = dataset.table(common.CONCEPT)
+    concept_ancestor_dest_table = dataset.table(common.CONCEPT_ANCESTOR)
+
+    descendants_table_ref = dataset.table(
+        MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE)
+    concept_table_ref = dataset.table(common.CONCEPT)
+    concept_ancestor_table_ref = dataset.table(common.CONCEPT_ANCESTOR)
+
+    # query will check the row counts of each table in the specified dataset
+    row_count_query = ROW_COUNT_QUERY.render(project_id=project_id,
+                                             dataset_id=dataset_id)
+
+    job = client.query(row_count_query)
+    results = job.result()
+
+    # will check if either the CONCEPT and CONCEPT_ANCESTOR tables is empty or not
+    # If so, the CONCEPT and CONCEPT_ANCESTOR tables will be copied from the common.VOCABULARY
+    # table to the destination dataset
+    if results.total_rows == 0:
+        client.copy_table(concept_source_table, concept_dest_table)
+        client.copy_table(concept_ancestor_source_table,
+                          concept_ancestor_dest_table)
+    else:
+        pass
+
+    # will check to see if CONCEPT table exists, will be copied from the CONCEPT table in the
+    # most recent VOCABULARY dataset if it is not found
+    try:
+        client.get_table(concept_table_ref)
+    except NotFound:
+        client.copy_table(concept_source_table, concept_dest_table)
+
+    # will check to see if CONCEPT_ANCESTOR table exists, will be copied from the CONCEPT_ANCESTOR table
+    # in the most recent VOCABULARY dataset if it is not found
+    try:
+        client.get_table(concept_ancestor_table_ref)
+    except NotFound:
+        client.copy_table(concept_ancestor_source_table,
+                          concept_ancestor_dest_table)
+
+    # will check to see if MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE exists, will be created if table is not found
+    try:
+        client.get_table(descendants_table_ref)
+    except NotFound:
+        bq.create_tables(client=client,
+                         project_id=project_id,
+                         fq_table_names=[descendants_table_name],
+                         exists_ok=False,
+                         fields=None)
```

Comment:
All of these can be removed, since you are already getting a data frame above, called `results`. This can be used for these tasks